object.js

```diff
@@ -0,0 +1,145 @@
+const data = {
+  projects: [
+    {
+      key: 0,
+      title: "Multi-Post Stories ",
+      image: "./image/Snapshoot portfolio.png",
+      description:
+        "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text..",
+      tags: ["html", "css", "ruby", "javaScript"],
+      liveSource: "#",
+      sourceCode: "#",
+      link: "#",
+    },
+    {
+      key: 1,
+      title: "Multi-Post Stories ",
+      image: "./image/Snapshoot portfolio.png",
+      description:
+        "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text..",
+      tags: ["html", "css", "ruby", "javaScript"],
```

Comment:
- [ ] Good job implementing this project requirement and dynamically rendering the popup window. There's a  little issue here tho. The image path you've used is wrong, and as a result, you have broken images throughout the page.
- [ ] This also brings up the issue with page accessibility, the images' `alt` attributes only show `image placeholder`, kindly replace it with a more descriptive placeholder.

_See images below._

![image](https://user-images.githubusercontent.com/49621279/123454215-2c006980-d595-11eb-9a82-f29c41364023.png)